bld/CLMBuildNamelist.pm

```diff
@@ -2682,12 +2679,13 @@ sub setup_logic_do_harvest {
       if (string_is_undef_or_empty($nl->get_value('flanduse_timeseries'))) {
          $cannot_be_true = "$var can only be set to true when running a transient case (flanduse_timeseries non-blank)";
       }
-      elsif (!&value_is_true($nl->get_value('use_cn'))) {
-         $cannot_be_true = "$var can only be set to true when running with CN (use_cn = true)";
-      }
-      elsif (&value_is_true($nl->get_value('use_fates'))) {
-         $cannot_be_true = "$var currently doesn't work with ED";
-      }
+      #cdk disabling this for now, but probably there is a better way to do it.
```

Comment:
This should be setup so that if use_cn AND use_fates are both false cannot_be_true would then be set.

Reply:
I think this is fixed now, @ekluzek please check to confirm, but otherwise I'll mark as resolved?